src/components/IdeaCardComponent.js

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import { CardActionArea, CardActions } from '@mui/material';
+import ShowMoreButton from './Buttons/ShowMore';
+
+function IdeaCard(props) {
+    const { idea } = props;
+
+    // Null checks
+    if (!idea) return null;
+
+    const description = idea.description.slice(0, 150); // Limit length of the description here
```

Comment:
Instead of leaving a comment, assign 150 to a new const and name it accordingly.

Plus you might want to create a new util function that take a description and returns a shorter version of it.

Also, take a look at how this can be done with CSS.

Reply:
I'll look through the CSS solutions. Maybe later implement something. Fixed